### PR TITLE
Feature 6

### PR DESCRIPTION
**Pull request by Namrata-J**
#7 here i have added ui for both cart page and wishlist page
have not added functionality yet!!
have also made them responsive.

Please review it, thankyou!!

Live preview: https://aprize-ecom-site.netlify.app/

### CODE REVIEW

File: src/components/Products-In-Cart/ProductsInCart.jsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                            </div>
+                        </div>
+                        <div className="ap_myCart-product-action-btns-section">
+                            <div className="ap_myCart-product-prize-info">
+                                <div className="ap_myCart-product-discountedPrice fw-4">₹1800</div>
+                                <div className="ap_myCart-product-originalPrice">₹2000</div>
+                            </div>
+                            <div className="ap_myCart-product-discount fw-4 a-tl">10% off</div>
+                            <div className="ap_myCart-product-quantity-details">
+                                <div className="ap_myCart-product-quantity-heading a-tl">Quantity:</div>
```
Comment (ShreyashMahajan) [inline src/components/Products-In-Cart/ProductsInCart.jsx:28]: classname="a-tl" can be more definable.

File: src/components/Products-In-Cart/productsInCart.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+.control-display-btn {
+    cursor: pointer;
+}
+
+.ap_myCart-product-action-btns {
+    display: flex;
+    flex-direction: column;
+    gap: 0.7rem;
```
Comment (ShreyashMahajan) [inline src/components/Products-In-Cart/productsInCart.css:107]: try to use px for less than 1rem

**Review by ShreyashMahajan — APPROVED** [review]
The code is well structured and easily understandable. 
Good to merge

File: src/components/Wishlisted-Product/WishlistedProducts.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                        <div className="product-description">
+                            <div className="card-title fw-4 a-tl">Nike</div>
+                            <div className="card-subtitle a-tl">Comfortable shoes for men</div>
+                        </div>
+                        <div className="card-product-price fw-3">
+                            <div className="ap_card-pp a-tl">
+                                <span className="p-mrp fw-1 a-tl">1000</span>
+                                <span className="p-discounted-price a-tl">₹900</span>
+                            </div>
+                            <div className="p-dicount a-tl">90% off</div>
```
Comment (Tanay-J) [inline src/components/Wishlisted-Product/WishlistedProducts.jsx:51]: Probably a typo in className

**Review by Tanay-J — APPROVED** [review]
Approving with a few suggestions:
- Naming scheme for classnames can be improved to make it more readable
- Using utility classes for commonly used properties like ```padding```, ```margin```, ```gap```  will improve code readability and reusability

File: src/components/Wishlisted-Product/WishlistedProducts.jsx (hunk 3 above)
Comment (Namrata-J) [reply src/components/Wishlisted-Product/WishlistedProducts.jsx:51]: thanks for pointing it out!!
didn't notice that mistake!!
would correct it!!

**Comment by Namrata-J** [comment]
> Approving with a few suggestions:
> 
> * Naming scheme for classnames can be improved to make it more readable
> * Using utility classes for commonly used properties like `padding`, `margin`, `gap`  will improve code readability and reusability

okk will work on that as well

**Pull request merged**